### PR TITLE
registratrion, login, salon list added

### PR DESCRIPTION
**Pull request by dnhirapara**

### CODE REVIEW

File: SalonCheckIn-frontend/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import Home from './Home';
+import Registration from './Registration';
+import Login from './Login';
+import ShopProfile from './ShopProfile';
+import CustomerProfile from './CustomerProfile';
+import Navbar from './Navbar';
+import Footer from './Footer';
+import RegisterCustomer from './RegisterCustomer';
+import RegisterSalon from './RegisterSalon';
+import ShopList from './ShopList';
```
Comment (dnhirapara) [inline SalonCheckIn-frontend/src/App.jsx:15]: make different directory with appropriate name for each related component.

**Review by dnhirapara — COMMENTED** [review]
fix imports.

File: SalonCheckIn-frontend/src/App.jsx (hunk 1 above)
Comment (Bjbrother) [reply SalonCheckIn-frontend/src/App.jsx:15]: okay,darshik

**Review by rishildhuwad — APPROVED** [review]
all checked

**Review by Bjbrother — APPROVED** [review]
checked all files

**Pull request merged**